packages/components/src/tooltip/stories/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { text } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import Tooltip from '../';
+import Button from '../../button';
+
+export default {
+	title: 'Components/ToolTip',
+	component: Tooltip,
+};
+
+export const _default = () => {
+	const tooltipText = text( 'Text', 'More information' );
+	return (
+		<Tooltip text={ tooltipText }>
+			<Button isSecondary>Hover for more information</Button>
```

Comment:
I wonder if we should use a Button in the example or just some random div. The reason is the Button already supports a "tooltip" prop (and label) which should have the same behavior.

Reply:
Yeah, that makes sense, should I update the readme too? That's where I usually pull the examples from.

Reply:
Yes, why not